sklearn/ensemble/forest.py

```diff
@@ -795,8 +795,8 @@ class RandomForestClassifier(ForestClassifier):
           number of samples for each node.
 
     min_weight_fraction_leaf : float, optional (default=0.)
-        The minimum weighted fraction of the input samples required to be at a
-        leaf node.
+        The minimum weighted fraction of the sum total of weights (of all
+        the input samples) required to be at a leaf node.
```

Comment:
It it worth noting, "Samples have equal weight when `sample_weight` is not provided, but `min_samples_leaf` is more efficient."